tor_ocr/main.py

```diff
@@ -76,8 +97,34 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+def ocr_space_url(url,
+                  overlay=False,
+                  api_key=os.getenv('OCR_API_KEY', 'helloworld')):
```

Comment:
What I've done on other projects when using environment variables is assign it to a python variable for readability and referencing the value of it concisely (in this case, it would help with line length).

```python
__OCR_API_KEY__ = os.getenv('OCR_API_KEY')


def ocr_space_url(url, overlay=False, api_key=__OCR_API_KEY__):
    # [...]
```

This is along the same lines as a previous comment of consistent interface with resources outside of the python program. We want one point in the program to map the internal and external data structures like this.

-----

On another note, I'd suggest not putting in a default value and letting it be `None` if the environment variable isn't set correctly. Then, in the program, we can confirm it's actually set and throw an error with customized messaging if it's not.

Reply:
Sounds good, will add tomorrow. However, the `helloworld` api key lets you do very few api tests, usually for this sort of thing so I'll probably leave that.

Reply:
Oh I must have missed that the hello world value is a legitimate testing key! In that case we could use it in automated tests (maybe leave a comment that it’s a valid test key?) when we circle back for that.

Reply:
Interestingly, it's not documented anywhere besides the examples. I would recommend against using it for testing, (simply because it's unknown what's going to happen if we overuse it) and just use it for development, i.e. in the REPL.